samples/memfs/memfs_test.go

```diff
@@ -1611,6 +1613,83 @@ func (t *MemFSTest) RenameNonExistentFile() {
 	ExpectThat(err, Error(HasSubstr("no such file")))
 }
 
+func (t *MemFSTest) GetListNoXAttr() {
+	var err error
+
+	// Create a file
+	filePath := path.Join(t.Dir, "foo")
+	err = ioutil.WriteFile(filePath, []byte("taco"), 0400)
+	AssertEq(nil, err)
+
+	names, err := xattr.List(filePath)
+	AssertEq(nil, err)
+	AssertEq(0, len(names))
+
+	_, err = xattr.Getxattr(filePath, "foo", nil)
+	AssertEq(fuse.ENOATTR, err)
+}
+
+func (t *MemFSTest) SetXAttr() {
+	var err error
+
+	// Create a file
+	filePath := path.Join(t.Dir, "foo")
+	err = ioutil.WriteFile(filePath, []byte("taco"), 0600)
+	AssertEq(nil, err)
+
+	err = xattr.Setxattr(filePath, "foo", []byte("bar"), 0x2)
```

Comment:
These magic constants (`0x2`, `0x1`) are confusing and hard to maintain. Please either upstream symbolic names for them into the `go-xattr` package, or add your own xattr package in this repo (probably in an `internal` directory) that has symbolic names for them.